templates/quoref/templates.yaml

```diff
@@ -0,0 +1,70 @@
+dataset: quoref
+templates:
+  6f1d5031-1377-4b8a-9475-987b2275b8da: !Template
+    id: 6f1d5031-1377-4b8a-9475-987b2275b8da
+    jinja: 'Given the following, context:
+
+      {{context}}
+
+      ===
+
+      answer the following question:
+
+      {{question}} |||
+
+      {{answers.text}}'
+    name: Template_1
+    reference: ''
+  aa26aab2-d2e7-4560-b7eb-0cbcff7c0f31: !Template
+    id: aa26aab2-d2e7-4560-b7eb-0cbcff7c0f31
+    jinja: '{{question}}
+
+      Answer based on the context below:
+
+      ===
+
+      {{context}} |||
+
+      {{answers.text}}'
+    name: Template_2
+    reference: ''
+  adf39ab3-dd5a-456f-aeaa-264453ef9339: !Template
+    id: adf39ab3-dd5a-456f-aeaa-264453ef9339
+    jinja: 'Given the context below:
+
+      {{context}}
+
+      ===
+
+      Predict the start index for the answer to the following question in the above
+      context!
+
+      {{question}} |||
+
+      {{answers.answer_start}}'
```

Comment:
When we tried this with T5 (predicting the index of the answer rather than the answer itself) it didn't really work (because of tokenization). I'd hesitate to have the model predict a location like this. Let me ask what people think in the slack.

Reply:
Sure, do let me know in case I would need to make any changes here.

Reply:
Leaving this as it is for the moment based on your confirmation yesterday.